### DISCUSSION TITLE
Mission statement

### DISCUSSION THREAD

**Discussion by ysooqe** [helix-editor/helix#7583] [General] [answered]
Hi there,

I am new to Helix and read through a lot of the website, the documentation as well as issues and discussions here on GitHub.
I have played around with it to see what it is and what it does. It looks like a very interesting project!

In a lot of discussions and issues here on GitHub, a few key points of the sentiment of the key developers towards Helix popped up here and there. Things like "Helix is not made to appeal to the masses, it just happens to align with what a lot of people want/need which is why it seems to be so popular at the moment" or things like "Helix development is not a democracy, we are only focussing on the things we [the Helix developers] want and need".

So now I was wondering:
What exactly are those things you [the developers] want for Helix? What do you envision it to be? What is the long-term outlook for the project? What is your mission statement?

I fully understand and respect the "we build a tool that fits our needs, it just happens to be popular but we won't do X or Y just for popularity". And of course there is not obligation whatsoever to publicly state or even define any of those things; a "mission statement" for others to read could even be somewhat contradictory to this.

But nevertheless, is there any chance you could write something like a "missions statement" down somewhere?
The reason why this would be great is that anyone discovering the editor would have a starting point for helping them decide if Helix is something they want to invest time in to learn it or if it might not be for them alltogether.
Maybe even just something along the lines of  "Helix is for you if: ..." and "Helix is not for you if: ..." could be highly appreciated aswell.

**Comment by pascalkuthe** [comment, marked as answer]
https://github.com/helix-editor/helix/blob/master/docs/vision.md

**Reply by ysooqe** [reply]
Thank you for the link! Maybe I have overlook this, but is there any reference to this from the official website or README?
